### PR TITLE
Update event-submission.md

### PR DESCRIPTION
**Pull request by SMotaal**
This proposes a `<!-- Annotated -->` variation of the templates with subtle formatting aiming for a linear visual flow to enhance accessibility.

[Split Diff](https://github.com/Nebrethar/Docs-Diversity-Inclusion-Badging/compare/feature-annotated-template?diff=split&expand=1)

[Source](https://github.com/Nebrethar/Docs-Diversity-Inclusion-Badging/blob/9f2062cf429618b12268880968a4ddbea46c938d/.github/ISSUE_TEMPLATE/event-submission.md#L1)

/cc @Nebrethar

### CODE REVIEW

**Comment by SMotaal** [comment]
@Nebrethar if something along those lines is okay, I can do the other templates.

**Comment by Nebrethar** [comment]
@SMotaal This looks great! Thank you. You are free to do the other templates.

File: .github/ISSUE_TEMPLATE/event-submission.md [hunk 1]
```diff
@@ -4,102 +4,156 @@ about: Submit an event to be badged
...
 ---
 
 # Event Submission
 
+<!--
+
 Thanks for starting the process of submitting an event for D&I Badging. To receive a CHAOSS D&I Event Badge, fill out this document as completely and honestly as possible.
 
-Items that are required for you to fill out are __bold__
+Items are marked with `Q` and `A` lines with comments for each answer.
```
Comment (Nebrethar) [inline .github/ISSUE_TEMPLATE/event-submission.md]: I think this is a good idea.

File: .github/ISSUE_TEMPLATE/event-submission.md [hunk 2]
```diff
@@ -4,102 +4,156 @@ about: Submit an event to be badged
...
-    * __Essay:__ 
-  
-Section 2:
-* Please provide a statement on the event's commitment to Attendee Diversity & Inclusion.
-  * __Essay:__ 
+### Section 1
+
+`Q` Are the attendee demographics of the event measured?
+
+`A` <!-- Yes/No (Required) -->
```
Comment (Nebrethar) [inline .github/ISSUE_TEMPLATE/event-submission.md]: Adding comments will help give the submitting party some direction.

**Pull request merged**